data/alzueta_LMRR.yaml

```diff
@@ -0,0 +1,1808 @@
+description: |-
+  ************************************************************************
+  ************************************************************************
+  ******************* For further documentation of mechanism:  ***********
+  ******************* see references below and their citations ***********
+  ************************************************************************
+  ************************************************************************
+  Nitrogen subset
+  P. Glarborg, J.A. Miller, B. Ruscic, S.J. Klippenstein,
+  "Modeling Nitrogen Chemistry in Combustion",
+  Prog. Energy Combust. Sci. 67 (2018) 31-68.
+  Updates marked by #
+
+generator: ck2yaml
+input-files: [glarborg2023-mech.txt, glarborg2023-therm.txt, glarborg2023-tran.txt]
+cantera-version: 2.6.0
+date: Tue, 07 Nov 2023 18:48:43 -0500
+
+units: {length: cm, time: s, quantity: mol, activation-energy: cal/mol}
+
+phases:
+- name: gas
+  thermo: ideal-gas
+  elements: [O, H, C, N, He, Ar]
+  species: [NH3, NO2, NO, N2O, H2, O2, O3, H, O, OH, HO2, H2O, H2O2, CO,
+    CO2, HOCO, CH2O, HCO, NH2, NH, N, NNH, N2H4, N2H3, tHNNH, cHNNH, H2NN,
+    NH2OH, H2NO, HNOH, HNO, HON, HONO, HNO2, NO3, HONO2, H2NCO, HNCO, NCO,
+    AR, HE, N2]
```

Comment:
It appears that there aren't any differences in species between the two versions of the file; in this case, you can actually import things from`alzueta.yaml` and shorten this file by around 900 lines (which would also emphasize what you are changing). For an example, have a look at `gri30_ion.yaml`.